src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -41,6 +96,40 @@ function ScrollDown(props) {
   );
 }
 
+function RetrieveStats() {
+  const { telescopeUrl } = useSiteMetadata();
+  const [stats, setStats] = useState({ stats: { posts: 0, authors: 0, words: 0 } });
+
+  useEffect(() => {
+    async function getStats() {
+      try {
+        const response = await fetch(`${telescopeUrl}/stats/year`);
+        if (response.status !== 200) {
+          throw new Error(response.statusText);
+        }
+
+        const stat = await response.json();
+        const localeStats = {
+          posts: stat.posts.toLocaleString(),
+          authors: stat.authors.toLocaleString(),
+          words: stat.words.toLocaleString(),
+        };
+        setStats(localeStats);
+      } catch (error) {
+        console.error('Error getting user info', error);
+      }
+    }
+
+    getStats();
+  }, [telescopeUrl]);
+
+  return (
+    <div className="stats">
+      This year, {stats.authors} of us wrote {stats.words} words and counting!
```

Comment:
The comma after year isn't needed.  I'd also consider changing this to:

`This year 70 of us have written 43,470 words and counting. Add yours!`